schema/data/dev/003_cif_project.sql

```diff
@@ -1,21 +1,29 @@
 begin;
 
-
 do $$
   begin
-    for project_id in 1..50 loop
-      insert into cif.project(operator_id, funding_stream_rfp_id, project_status_id, proposal_reference, summary, project_name, total_funding_request) values
+    for index in 1..50 loop
+      insert into cif.form_change(
```

Comment:
This is closer to how projects are created when interacting with the application, but not quite the same. If this allows to edit existing projects it's good enough, but this may cause issues in the future. Using `cif.create_project()` would probably be the way to mirror the application's behaviour

Reply:
Leaving as is for now - we can open up a tech debt card if needed.  Editing projects seems to work fine.